### ISSUE TITLE
Semantic Kernel Upgrading

### ISSUE THREAD

**Issue by vladborys** [Azure/gpt-rag-orchestrator#106]
https://github.com/microsoft/semantic-kernel version 1.13

https://devblogs.microsoft.com/semantic-kernel/introducing-agents-in-semantic-kernel/

**Comment by placerda** [comment]
@vladborys Semanti Kernel is already updated in requirements.txt, should we close this one?

**Comment by vladborys** [comment]
@placerda, Yes, there was a pull request with SK update. I reviewed, tested and approved it from my end. So, this task can be closed.
